rewrite-core/src/main/java/org/eclipse/jgit/util/FS.java

```diff
@@ -852,16 +852,16 @@ public static FS detect() {
 	/**
 	 * Whether FileStore attributes should be determined asynchronously
 	 *
-	 * @param asynch
+	 * @param async
 	 *            whether FileStore attributes should be determined
 	 *            asynchronously. If false access to cached attributes may block
 	 *            for some seconds for the first call per FileStore
 	 * @since 5.1.9
 	 * @deprecated Use {@link FileStoreAttributes#setBackground} instead
 	 */
 	@Deprecated
-	public static void setAsyncFileStoreAttributes(boolean asynch) {
-		FileStoreAttributes.setBackground(asynch);
+	public static void setAsyncFileStoreAttributes(boolean async) {
+		FileStoreAttributes.setBackground(async);
```

Comment:
Slightly surprised by this one; would have expected there was a reason we had `asynch` previously; we'll see how the tests go.

Thanks for this massive effort btw! Out of curiosity: how much of this is automated versus manual tweaks?